CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 - match: extend OS characteristic to match OS_ANY to all supported OSes #1324 @mike-hunhoff
 - explorer: fix exception when plugin loaded in IDA hosted under idat #1341 @mike-hunhoff
 - extractor: fix IDA and vivisect string and bytes features overlap and tests #1327 #1336 @xusheng6 
+- explorer: improve embedded PE detection #1344 @mike-hunhoff
```

Comment:
there's also the explorer section just below for these

Reply:
nit: maybe describing the improvement would be even more useful to an interested reader.

Reply:
```suggestion
- explorer: improve embedded PE detection performance and reduce FP potential #1344 @mike-hunhoff
```